sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient.cs

```diff
@@ -0,0 +1,969 @@
+                return Response.FromValue(response.Status == 200, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Check if the connection with the given connectionId exists</summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response<bool> GroupExists(string group, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CheckGroupExistence");
+            scope.Start();
+            try
+            {
+                var response = RestClient.GroupExists(hub, group, cancellationToken);
+                return Response.FromValue(response.Status == 200, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Check if there are any client connections connected for the given user</summary>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.</returns>
+        public virtual async Task<Response<bool>> UserExistsAsync(string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CheckUserExistenceAsync");
+            scope.Start();
+            try
+            {
+                var response = await RestClient.UserExistsAsync(hub, userId, cancellationToken).ConfigureAwait(false);
+                return Response.FromValue(response.Status == 200, response);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary>Check if there are any client connections connected for the given user</summary>
+        /// <param name="userId">Target user Id</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> representing the result of the operation.</returns>
+        public virtual Response<bool> UserExists(string userId, CancellationToken cancellationToken = default)
+        {
+            using var scope = _clientDiagnostics.CreateScope("WebPubSubServiceRestClient.CheckUserExistence");
+            scope.Start();
+            try
+            {
+                var response = RestClient.UserExists(hub, userId, cancellationToken);
+                return Response.FromValue(response.Status == 200, response);
```

Comment:
Consider a helper to wrap this logic so we've got one place to update to `< 300` if that ever becomes necessary.